CDCgov/prime-reportstream
# 12413: PV1/PV2 to Encounter mapping

This PR adds HL7->FHIR->HL7 mappings for PV1/PV2 <-> Encounter

Should be reviewed with https://github.com/CDCgov/hl7v2-fhir-converter/pull/6

Test Steps:
1. Verify the mapping inventory
  - https://docs.google.com/spreadsheets/d/1Sqt8f_LpBVMIdJxcAlB4emDOBqXV8rhy60V2_yaDp7c/edit#gid=0
  - https://docs.google.com/spreadsheets/d/1ILbJ8GmvNdEW4XmvrmEHtZxsRM1bk5hHl5WD2oDSa4Q/edit#gid=0
2. Run the tests

## Changes
- Adds the mapping for PV1/PV2 - Encoutner

## Checklist

### Testing

### Process

## Linked Issues
- #12413 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
